portfolio/src/main/webapp/script.js

```diff
@@ -16,13 +16,12 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  const greetings = "Bring your passion and whole self to work everyday!"
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  //const greeting = greetings[Math.floor(Math.random() * greetings.length)];
```

Comment:
Remove this line if not required. Comments shouldn't have any unnecessary codes.